mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/api/akka/route/Jackson.java

```diff
@@ -39,7 +40,8 @@ public class Jackson {
     private static final ObjectMapper defaultObjectMapper = new ObjectMapper()
             .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
             .configure(SerializationFeature.FAIL_ON_EMPTY_BEANS, false)
-            .registerModule(new Jdk8Module());
+            .registerModule(new Jdk8Module())
+            .registerModule(new JavaTimeModule());
```

Comment:
This has some implications, I think. For example, with this change, timestamps would be serialized differently, which may not be what our clients expect. Would it be possible to avoid this?

Reply:
Are you sure this is the case?

Reply:
There are tests where we serialize and deserialize timestamps we should break if this new module had implications, no?

Reply:
I prefer to avoid this change if possible. I'm not so sure about the coverage of the tests themselves.

Reply:
I think we have the following options:
- leave as is but we risk to make unwanted changes to other ser/deser of timestamps.
- use separate object mapper which registers `JavaTimeModule` 
- Change `Instant` to long

I prefer option 2, thoughts?

Reply:
The JsonSerializer already registers JavaTimeModule, I think. So I would prefer we use that. I'm trying to avoid new ObjectMapper initialization precisely because of this reason.